packages/app/src/screens/RoutineEditScreen.tsx

```diff
@@ -26,17 +48,26 @@ const RoutineEditScreen = () => {
             <Text fontType='BOLD_16'>준비 항목 ({taskData.length})</Text>
           </View>
           {/* 히스토리 버튼 눌렀을 때 모달 창 */}
-          <Pressable style={{ flex: 1 }} onPress={() => {}}>
+          <Pressable
+            style={{ flex: 1 }}
+            onPress={() => {
+              toggleHistoryModal();
+            }}>
             <Text fontType='BOLD_12' color={colors.grayDark}>
               히스토리
             </Text>
           </Pressable>
         </View>
-        {taskData.map(() => (
-          <TaskItem duration={30} title='브런치 먹기' edit />
+
+        {taskData.map((task) => (
+          <TaskItem duration={task.time} title={task.name} type={'edit'} />
```

Comment:
```suggestion
          <TaskItem key={task.id} duration={task.time} title={task.name} type={'edit'} />
```

`key`값 넣어주는 거 잊지 마세용🤗